packages/react-components/src/components/icons/postgresql.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
```

Comment:
is this icon used anywhere?

Reply:
Same as above: https://github.com/questdb/saas-client/blob/master/modules/connection-details/badges.tsx#L7